### ISSUE TITLE
A11Y: EmojiList should be keyboard accessible

### ISSUE THREAD

**Issue by comp615** [missive/emoji-mart#218]
When focus is in the emoji picker search input (or the accessible), using the arrows should navigate the available emojis. 

In addition, the input and list should be part of a `role="combobox"` since they are similar to a typeahead.
